.github/workflows/smoke-check.yml

```
@@ -0,0 +1,46 @@
+name: Smoke check
+run-name: Smoke check - ${{ github.event_name == 'pull_request' && format('PR {0}', github.event.pull_request.number) || github.event_name == 'push' && 'Push' || format('Requested by @{0}', github.actor) }}
+
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)